README.md

```diff
@@ -221,6 +221,24 @@ Nuxeo AI Core provides 3 Java modules:
     </thresholdConfiguration>
 </extension>
 ```
+  * Continues Export - Once a week a Cron scheduler fires `startContinuesExport` event.
+  On such event the system will retrieve all AI Model defined under
+  configured project and perform evaluation of the current data to check if it needs to be uploaded.
+  ```xml
+  <extension target="org.nuxeo.ecm.core.scheduler.SchedulerService" point="schedule">
+    <schedule id="continues_export_default">
+      <eventId>startContinuesExport</eventId>
+      <!-- Every week at midnight -->
+      <cronExpression>0 0 0 1/7 * ?</cronExpression>
```

Comment:
I would check that `every 7 days starting on the 1st, every month` is not starting too late if the addon is loaded after the 1st :D